sessionctx/variable/sysvar.go

```diff
@@ -1883,4 +1890,6 @@ const (
 	RandSeed1 = "rand_seed1"
 	// RandSeed2 is the name of 'rand_seed2' system variable.
 	RandSeed2 = "rand_seed2"
+	// TiDBEnableConcurrencyDDL indicates whether to enable the new DDL framework.
+	TiDBEnableConcurrencyDDL = "tidb_enable_concurrency_ddl"
```

Comment:
Gramatically EnableConcurrentDDL makes more sense. Rest LGTM:
```suggestion
	// TiDBEnableConcurrentDDL indicates whether to enable the new DDL framework.
	TiDBEnableConcurrentDDL = "tidb_enable_concurrent_ddl"
```